{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -1,3 +1,54 @@
+[build-system]
+requires = ["setuptools>=62.0", "setuptools-git-versioning<2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "{{ cookiecutter.project_name }}"
+dynamic=['version']
+authors = [
+  { name="Simon J.L. Billinge group", email="simon.billinge@gmail.com" },
+]
+maintainers = [
+  { name="Simon J.L. Billinge group", email="simon.billinge@gmail.com" },
+]
+description = "{{ cookiecutter.project_short_description }}"
+keywords = []
+readme = "README.rst"
+requires-python = ">=3.9"
+classifiers = [
+        'Development Status :: 5 - Production/Stable',
+        'Environment :: Console',
+        'Intended Audience :: Developers',
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: MacOS :: MacOS X',
+        'Operating System :: Microsoft :: Windows',
+        'Operating System :: POSIX',
+        'Operating System :: Unix',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.9',
```

Comment:
I think these shouldn't be hard coded.  We may need some pretty fancy jinja to make it work, but I think hte cookiecutter asks a question about minimum python version, so if we select 3.10 or 3.11 or whatever, we want this to be built correctly.

Reply:
We could put a placeholder there and replace it with a post-gen hook, using a new cookiecutter variable (`min_version`, maybe?)  we define.

Reply:
THere is already a question to the user on min version, the question is how to code it here.

Reply:
I don't want to over-rely on post-gen hooks. Can't we use Jinja2 syntax to do this using a for loop?

We should also theoretically have a maximum supported python version in our `cookiecutter.json`.

Reply:
I'm taking a look at this right now, it definitely seems like there's a way to do it with Jinja2.